### ISSUE TITLE
[Feature Request] Return of the MKS Ventral drills

### ISSUE THREAD

**Issue by Deadpan110** [UmbraSpaceIndustries/MKS#267]
With the current radial drills and the upcoming inline drills, there are a few USI fanatics that remember and really miss the Ventral drill assembles.

They go hand in hand along side the Karbonite Ventral drill and as such, are perfect for use within the LS-40 and LSX landing frames.

The Karbonite Ventral Drill also makes its way on to the belly of my smaller HoneyBadger designs which free the sides up for wheels and landing legs - with the reintroduction of MKS Ventral drills, it could be possible to make a fleet of them, each with their own resources to collect (a matching set colour coded with FTT's texture select).

TLDR; We miss the MKS Ventral drill assemblies :)

Ref: [USI-Kolonization-Systems - Forum Post 4542](http://forum.kerbalspaceprogram.com/threads/79588-0-25-USI-Kolonization-Systems-%28MKS-OKS%29-%280-21-2%29-2014-10-07?p=1509480&viewfull=1#post1509480)
